docs/edit/skip-tests.md

```diff
@@ -0,0 +1,38 @@
+Three decorators allow you to skip test functions or classes for a library:
+
+* `@irrelevant`: The tested feature/behavior is irrelevant to the library, meaning the feature is either purposefully not supported by the lib or cannot reasonably be implemented
+* `@bug`: The lib does not implement the feature correctly/up to spec
+* `@flaky` (subclass of `bug`): The feature sometimes fails, sometimes passed, the test is skipped
+* `@missing_feature`: The tested feature/behavior does not exist in the library or there is a deficit in the test library that blocks this test from executing for the lib
+
+To skip specific test functions within a test class, use them as in-line decorators (Example below). 
+To skip test classes or test files, use the decorator in the library's [manifest file](./manifest.md).
+
+The decorators take several arguments:
+
+* `condition`: boolean, tell if it's relevant or not. As it's the first argument, you can omit the arguement name
+* `library`: provide library. version numbers are allowed (`java@1.2.4`)
```

Comment:
Not sure where to place this but should we mention setting the version to `X.X.0.dev` will ensure tests are run against unreleased changes (assuming x.x.0 refers to the next minor release).

Reply:
I think we indirectly mention this in version.md but I think we can be more clear,